tests/components/person/test_init.py

```diff
@@ -44,7 +46,8 @@ def storage_setup(hass, hass_storage, hass_admin_user):
 async def test_minimal_setup(hass):
     """Test minimal config with only name."""
     config = {DOMAIN: {'id': '1234', 'name': 'test person'}}
-    assert await async_setup_component(hass, DOMAIN, config)
+    with assert_setup_component(1):
```

Comment:
What does the count number represent in this case? Is it number of set up persons (from config)?

Reply:
I don't request any changes. Not sure what happened there. This is a just a question for my curiosity.

Reply:
Yes, it will be the amount of persons. Initially it was for valid platforms & should make debugging of especially validators a lot easier